lsp/server/src/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+export { TempProjectDirManager } from './test/TestHelper';
```

Comment:
It used to be in root project, moved here for broader usage

Reply:
Can we improve on this? We are importing a module used to test into production code. If `TempProjectManager` is needed to be used in both for production and test then it should not be in `TestHelper`.